code/modules/power/tesla/coil.dm

```diff
@@ -31,7 +31,7 @@
 	for(var/obj/item/stock_parts/capacitor/C in component_parts)
 		power_multiplier += C.rating
 		zap_cooldown -= (C.rating * 20)
-	input_power_multiplier = power_multiplier
+	input_power_multiplier = (0.85 * (power_multiplier / 4)) //Max out at 85% efficency.
```

Comment:
```suggestion
	input_power_multiplier = power_multiplier * 0.2125 //Max out at 85% efficency.
```

Reply:
It's an operation faster but a lot less clear. I don't think I want to do this